xml/vk.xml

```diff
@@ -2561,6 +2561,55 @@ typedef void <name>CAMetalLayer</name>;
             <member><type>float</type>   <name>x</name></member>
             <member><type>float</type>   <name>y</name></member>
         </type>
+        <type category="struct" name="VkSwapchainTimingPropertiesEXT">
+            <member values="VK_STRUCTURE_TYPE_SWAPCHAIN_TIMING_PROPERTIES_EXT"><type>VkStructureType</type> <name>sType</name></member>
+            <member>const <type>void</type>*                      <name>pNext</name></member>
+            <member><type>uint64_t</type>                         <name>refreshDuration</name><comment>Number of nanoseconds from the start of one refresh cycle to the next</comment></member>
+            <member><type>VkBool32</type>                         <name>variableRefresh</name></member>
+        </type>
+        <type category="struct" name="VkSwapchainTimeDomainPropertiesEXT">
+            <member values="VK_STRUCTURE_TYPE_SWAPCHAIN_TIME_DOMAIN_PROPERTIES_EXT"><type>VkStructureType</type> <name>sType</name></member>
+            <member><type>void</type>*                            <name>pNext</name></member>
+            <member><type>VkTimeDomainEXT</type>                  <name>timeDomain</name><comment>Available time domain to use with the swapchain</comment></member>
+        </type>
+        <type category="struct" name="VkSwapchainTimingInfoEXT">
+            <member values="VK_STRUCTURE_TYPE_SWAPCHAIN_TIMING_INFO_EXT"><type>VkStructureType</type> <name>sType</name></member>
+            <member>const <type>void</type>*                      <name>pNext</name></member>
+            <member><type>VkTimeDomainEXT</type>                  <name>timeDomain</name><comment>Available time domain to use with the swapchain</comment></member>
```

Comment:
Nit: maybe we could tweak the wording a little here to better reflect the intent. The fact that the definition is exactly the same as `VkSwapchainTimeDomainPropertiesEXT` is a little bit confusing.

What about something among those lines: "Time domain selected by the application to be used for the swapchain time values"?

Reply:
Fixed, thank you.